deploy/apecloud-mysql-cluster/values.yaml

```diff
@@ -2,66 +2,58 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 3
+## @param version MySQL cluster version
+##
+version: ac-mysql-8.0.30
 
-terminationPolicy: Delete
+## @param mode MySQL cluster topology mode, standalone, replication or raftGroup
+##
+mode: standalone
 
-clusterVersionOverride: ""
-nameOverride: ""
-fullnameOverride: ""
+## @param replicas MySQL cluster replicas
+## for standalone mode, replicas can not be changed, it is always 1
+## for replication mode, replicas can not be changed, it is always 2
+## for raftGroup mode, replicas can be changed, the default value is 3
+##
+replicas: 1
 
-monitor:
-  enabled: false
+## @param cpu
+##
+cpu: 0.5
 
-resources: { }
-  # We usually recommend not to specify default resources and to leave this as a conscious
-  # choice for the user. This also increases chances charts run on environments with little
-  # resources, such as Minikube. If you do want to specify resources, uncomment the following
-  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+## @param memory, the unit is Gi
+##
+memory: 1
 
-  # limits:
-  #   cpu: 500m
-  #   memory: 2Gi
-  # requests:
-  #   cpu: 100m
-  #   memory: 1Gi 
+## @param storage size, the unit is Gi
+##
+storage: 20
 
+## @param monitoring interval, 0 is no monitoring
+##
+monitoringInterval: 0
 
-logVolume:
-  # Provide sizeLimit to emptyDir volume.
-  sizeLimit:
-  # Provide a medium for emptyDir volumes.
-  medium:
+## @param terminationPolicy
+terminationPolicy: Delete
 
-persistence:
-  enabled: true
-  data:
-    storageClassName:
-    size: 1Gi
-  # separate log volume for WAL log
-  wal:
-    enabled: false
-    storageClassName:
-    size: 1Gi
-  # separate log volume for app log
-  log:
-    # If enabled, log emptyDir volume will be superseded.
-    enabled: false
-    storageClassName:
-    size: 1Gi
+## @param proxyEnabled if true, enable the proxy
+## if mode is standalone or replication, proxyEnabled is always false
+## if mode is raftGroup, proxyEnabled can be true or false
+##
+proxyEnabled: false
 
-enabledLogs:
-  - slow
-  - error
+## @param availablePolicy
+##
+availabilityPolicy: none
 
-topologyKeys:
-  - kubernetes.io/hostname
+## @param tenancy dedicated or shared
+##
+tenancy: SharedNode
 
-## @param tolerations
-## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+## @param hostNetworkAccessible if true, expose the vpc service
 ##
-tolerations: [ ]
+hostNetworkAccessible: false
 
-# The RBAC permission used by cluster component pod, now include event.create
-serviceAccount:
-  name: ""
+## @param publiclyAccessible if true, expose the internet service
+##
+publiclyAccessible: false
```

Comment:
how about placing following common values in cluster-libchart?

```
replicas
cpu
memory
storage
monitoringInterval
terminationPolicy
availabilityPolicy
tenancy
hostNetworkAccessible
publiclyAccessible
```

Reply:
I tried it but the overriding and rendering behaviour of values between parent chart and child chart is a bit confusing, some fields cannot render properly. I plan to revisit this after integrating more other cluster helm charts, don't want to solve this in the current PR.